src/js/components/DataTable/doc.js

```diff
@@ -393,6 +394,12 @@ export const themeDoc = {
     type: 'string | object',
     defaultValue: undefined,
   },
+  'dataTable.header.units': {
+    description: `The styles for the units display in the header. This
+    can be any valid Text properties`,
+    type: 'object',
+    defaultValue: '{ size: "xsmall" }',
```

Comment:
Since you are defining the default properties of the theme, I would have expected to see this default value on base.js and not hardcode on the component file.

Reply:
Perfect. Done